.changeset/heavy-dingos-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/firestore': minor
```

Comment:
I think this should just be a patch. Otherwise LGTM.

Reply:
I agree, should be a patch.

Reply:
Changed